HOWTO.dual.softhsm2

```diff
@@ -0,0 +1,85 @@
+############################################################################
+#
+# SoftHSMv2 dual versions
+#
+# Works with CentOS 7
+#
+# The opensc rpm needs to be installed.
+#
+############################################################################
+#
+# Install a software HSM module for doing the PKCS11 testing of libcoap.
+# Real hardware can be used if you have the appropriate library module.
+#
+# It is assumed that the opensc rpm is installed.
+#
+# There is a softhsm2 rpm, but it uses openssl prior to 1.1.0. This may
```

Comment:
I recommend a more future-proof statement such as: When installing softhsm2 from your system's package manager, check that the openssl version used by this version is at least...
Pretty much what is said in the paragraph below.

Reply:
Updated with
````
# When installing SoftHSMv2 from your system's package manager, check that the
# OpenSSL version is at least 1.1.0.  If not, then you will need to dual 
# install SoftHSMv2 as below; otherwise you can use the existing SoftHSMv2.
#
# SoftHSMv2 is built using OpenSSL, but GnuTLS can use the PKCS11 interface.
# Note that if the default OpenSSL version is less than 1.1.0, you will need
# to install a dual stack version of OpenSSL as per HOWTO.dual.openssl.
````